CMake/BuildPeleCExe.cmake

```diff
@@ -21,6 +21,45 @@ function(build_pelec_exe pelec_exe_name pelec_lib_name)
   target_include_directories(${pelec_exe_name} PRIVATE ${CMAKE_BINARY_DIR})
   target_include_directories(${pelec_exe_name} PRIVATE ${CMAKE_SOURCE_DIR}/Source/Params/param_includes)
 
+  # Set PeleMP flags
+  set(PELEMP_SRC_DIR ${CMAKE_SOURCE_DIR}/Submodules/PeleMP/Source)
+  if(PELEC_ENABLE_AMREX_PARTICLES AND PELEMP_SPRAY_FUEL_NUM GREATER 0)
```

Comment:
Should `PELEC_ENABLE_AMREX_PARTICLES` be `PELEC_ENABLE_SPRAY` ?

Reply:
Well, I wanted to leave open the possibility that AMReX particles might be implemented and used in PeleC that aren't spray particles. So if AMReX particles are turned on and `SPRAY_FUEL_NUM` is set, it will know that spray physics are being used.

Reply:
Makes sense!